llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -10621,12 +10621,64 @@ bool SIInstrInfo::analyzeCompare(const MachineInstr &MI, Register &SrcReg,
   return false;
 }
 
+bool SIInstrInfo::invertSCCUse(MachineInstr *SCCDef) const {
```

Comment:
Needs a function comment explaining what it does and what the return value means.